dkeyserver/App/main.cpp

```diff
@@ -22,6 +23,12 @@ char s_port[] = "8888";
 sgx_enclave_id_t g_enclave_id;
 
 using namespace std;
+std::string target_ip_addr;
+uint16_t target_port = 0;
+static const char *_sopts = "i:p:";
+static const struct option _lopts[] = {{"ip", required_argument, NULL, 'i'},
+                                       {"port", required_argument, NULL, 'p'},
```

Comment:
set the two as optional parameters, and add -h as optional help.

Reply:
done.